### DISCUSSION TITLE
Desktop Security Onion issue

### DISCUSSION THREAD

**Discussion by SSLmike** [Security-Onion-Solutions/securityonion#2065] [Unsupported Versions] [answered]
Hi everyone, I'm new to using Security Onion and tried to install it but failing to get the CentOS desktop as well, I can only get the command line.  I tried some settings during the installation but the result doesn't change.  If anyone knows what I have to select to have it solves a problem thanks.

**Comment by TOoSmOotH** [comment, marked as answer]
run so-analyst-install after you install SO.

**Reply by SSLmike** [reply]
Thank you so much I've been looking for a way to get the desktop for a while.
If I can ask one more thing, I would like to know if you can have the same interface for both sniffing and monitoring and if so how to do it.
Thanks again for the support.

**Reply by TOoSmOotH** [reply]
you need 2 interfaces unless you are running just import mode
